src/includes/getting-started-install/apple.ios.mdx

```diff
@@ -11,4 +11,6 @@ target 'YourApp' do
 end
 ```
 
-Afterwards run `pod install`.
+Then run `pod install`.
+
+We also support alternate [installation methods](install/).
```

Comment:
I wonder how often people use cocoapods vs SPM but I suspect SPM might become the most popular approach soon if it's not yet.

Can we at least mention it here somehow like?

```suggestion
We also support alternate [installation methods such as Swift Package Manager](install/).
```

Reply:
We'll keep it simple with the link, but I can track in Google Analytics the number of times Cocoa is viewed v. SPM.